charts/model-engine/values_circleci.yaml

```diff
@@ -270,12 +270,36 @@ recommendedHardware:
       cpus: 80
       gpus: 8
       memory: 800Gi
-      storage: 460Gi
+      storage: 640Gi
       gpu_type: nvidia-hopper-h100
   byModelName:
     - name: llama-3-8b-instruct-262k
       cpus: 20
       gpus: 2
       memory: 40Gi
       storage: 40Gi
+      gpu_type: nvidia-hopper-h100
+    - name: deepseek-coder-v2-lite
+      cpus: 160
+      gpus: 8
+      memory: 800Gi
+      storage: 640Gi
+      gpu_type: nvidia-hopper-h100
```

Comment:
i think this is optional and we don't need them here. if we intend to add them, `deepseek-coder-v2-lite` doesn't need so much resource

Reply:
oh, i was just looking at the instructions on their repo: https://huggingface.co/deepseek-ai/DeepSeek-Coder-V2-Lite-Instruct#5-how-to-run-locally

Reply:
they just said "If you want to utilize DeepSeek-Coder-V2 in BF16 format for inference, 80GB*8 GPUs are required" right? i think 8x80GB only applies to DeepSeek-Coder-V2 not lite